src/main/resources/io/viash/platforms/nextflow/WorkflowHelper.nf

```diff
@@ -603,10 +606,467 @@ def paramsToList(params, config) {
 }
 
 def paramsToChannel(params, config) {
+  System.err.println("Warning: paramsToChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   Channel.fromList(paramsToList(params, config))
 }
 
 def viashChannel(params, config) {
+  System.err.println("Warning: viashChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   paramsToChannel(params, config)
     | map{tup -> [tup.id, tup]}
 }
+
+/**
+ * From a viash config, gather the config entries 
+ * for arguments that have property 'multiple: true'
+ */
```

Comment:
`/**` signifies javadoc, but a title and the `@param config` and `@return` entries are missing. Can you add them?